packages/cf-style-container/package.json

```diff
@@ -12,9 +12,10 @@
   },
   "dependencies": {
     "fela": "^5.0.0",
-    "merge-options": "0.0.64",
+    "merge-options": "^1.0.0",
+    "prop-types": "^15.5.8",
     "react-fela": "^5.0.0",
-    "prop-types": "^15.5.8"
+    "underscore.string": "^3.3.4"
```

Comment:
This is often needed in construct the names of some of our existing CSS rules like `tableStripedDefaultBackground`. Since underscore.string comes with tons of functions, it's too easy to import the entire library and prevent webpack from treeshaking well. It's better to not tell people we are using underscore.string. If you need to capitalize, just use the one in cf-style-container. This makes it easier to swap out underscore.string too.